src/LogicTasks/Semantics/Decide.hs

```diff
@@ -140,28 +141,7 @@ start :: [Int]
 start = []
 
 partialGrade :: OutputCapable m =>  DecideInst -> [Int] -> LangM m
-partialGrade DecideInst{..} sol = do
-  preventWithHint (null sol)
-    (translate $ do
-      german "Lösung enthält Indizes?"
-      english "Solution contains indices?"
-    )
-    (translate $ do
-      german "Die Lösung muss mindestens einen Index enthalten."
-      english "The solution must contain at least one index."
-    )
-
-  preventWithHint (any (\x -> x < 1 || x > tableLen) sol)
-    (translate $ do
-      german "Lösung enthält nur gültige Indizes?"
-      english "Solution only contains valid indices?"
-    )
-    (translate $ do
-      german "Die Lösung enthält mindestens einen ungültigen Index."
-      english "The solution contains at least one invalid index."
-    )
-
-  pure ()
+partialGrade DecideInst{..} sol = multipleChoiceSyntax False [1..tableLen] $ nubOrd sol
```

Comment:
Why is the `nubOrd` needed here?

Or more to the point, should I be worried that the other applications of `multipleChoiceSyntax` in the repo to not take care to first `nub` the student submission?

Reply:
According to the definition of `multipleChoiceSyntax` (https://github.com/fmidue/output-blocks/blob/cc5e88053b905827221a736355b6c0678c6ed6cd/output-blocks/src/Control/OutputCapable/Blocks.hs#L183-L184) the `nubOrd` should not be necessary here.